app/sql.js

```diff
@@ -559,11 +561,42 @@ async function removeAllIdentityKeys() {
 
 const PRE_KEYS_TABLE = 'preKeys';
 async function createOrUpdatePreKey(data) {
-  return createOrUpdate(PRE_KEYS_TABLE, data);
+  const { id, recipient } = data;
+  if (!id) {
+    throw new Error('createOrUpdate: Provided data did not have a truthy id');
+  }
+
+  await db.run(
+    `INSERT OR REPLACE INTO ${PRE_KEYS_TABLE} (
+      id,
+      recipient,
+      json
+    ) values (
```

Comment:
Capital "VALUES" ?

Reply:
I'm just keeping it consistent to what was already in the original signal file, thus you'll see all of them lowercase `values`

Reply:
Ok for consistency with the original file, even if that one is not consistent 😛